lottie/src/main/java/com/airbnb/lottie/model/layer/ImageLayer.java

```diff
@@ -107,7 +107,10 @@ public class ImageLayer extends BaseLayer {
       if (lottieDrawable.getMaintainOriginalImageBounds()) {
         outBounds.set(0, 0, lottieImageAsset.getWidth() * scale, lottieImageAsset.getHeight() * scale);
       } else {
-        outBounds.set(0, 0, getBitmap().getWidth() * scale, getBitmap().getHeight() * scale);
+        Bitmap bitmap = getBitmap();
+        if (bitmap != null) {
```

Comment:
What happens if this is null? Will outBounds potentially have a stale value from another frame?

Reply:
@gpeal I don't have the full context here so maybe @geomaster can chime in on what scenarios would cause the bitmap to be null and whether we need alternate behavior around updating `outBounds`.

Reply:
Unfortunately I'm not sure why the bitmap would be null - presumably, it has not loaded yet, or it didn't load correctly?

If the bitmap is null, we would be rendering nothing, so I think arguably the correct thing to do is simply to set to an empty rectangle, with `outBounds.set(0, 0, 0, 0)`. 

Thoughts?